.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,68 @@
+name: "Feature request"
+description: "Suggest an idea"
+body:
+  - type: markdown
+    attributes:
+      value: "Please fill in each section completely. Thank you!"
+  - type: textarea
```

Comment:
I just noticed that the issue template has this as an `input`; maybe that's better? Or, maybe the issue template should use a text area instead?

Reply:
bug_report.yml has this same field as `type: input`, while this one has it as `type: textarea`; I'm just trying to decide which one is better. I'm leaning towards both being `textarea`.

Reply:
Ah, yes, I noticed that. Yeah, I prefer that one, too.